pkg/ccl/storageccl/export.go

```diff
@@ -200,7 +203,11 @@ func evalExport(
 			// Create a unique int differently.
 			nodeID := cArgs.EvalCtx.NodeID()
 			exported.Path = fmt.Sprintf("%d.sst", builtins.GenerateUniqueInt(base.SQLInstanceID(nodeID)))
-			if err := exportStore.WriteFile(ctx, exported.Path, bytes.NewReader(data)); err != nil {
+			if err := retry.WithMaxAttempts(ctx, retry.Options{}, maxUploadRetries, func() error {
+				// We blindly retry any error here because we expect the caller to have
+				// verified the target is writable before sending ExportRequests for it.
+				return exportStore.WriteFile(ctx, exported.Path, bytes.NewReader(data))
```

Comment:
very optional: Worth logging? Maybe V=3 or something?

Reply:
good point. If i do it on err only, v=1 seems fine. added.